tests/Backend/CommonPart1/AlterTableTest.php

```diff
@@ -210,6 +210,97 @@ public function testTooManyColumns(): void
         }
     }
 
+    public function testTooManyColumnsCountFailed(): void
+    {
+        $this->skipTestForBackend([
+            self::BACKEND_EXASOL,
+            self::BACKEND_BIGQUERY,
+        ], 'Other backends does have other limits');
+
+        try {
+            $data = [
+                'name' => 'tooManyColumns',
+                'primaryKeysNames' => [],
+                'columns' => [],
+            ];
+            // generate table definition with too many columns - over limit on every backend
+            for ($i = 1; $i <= 5000; $i++) {
+                $data['columns'][] = [
+                    'name' => 'col' . $i,
+                    'definition' => [
+                        'type' => 'BYTEINT',
+                        'nullable' => true,
+                    ],
+                ];
+            }
+            $this->_client->createTableDefinition(
+                $this->getTestBucketId(),
+                $data,
+            );
+            $this->fail('There were 5000 columns man. fail.');
```

Comment:
co znamena to `man.` ?

Reply:
tos tam napsal v cervenci ty :-) Chvili jsem nad tim dumal, ale nejspis to bude "je tam 5000 sloupcu kámo. spadlo to."

Reply:
haha, tak z toho prosim udelej neco smysluplneho.

Reply:
upraveno